File: debutcourandroid/app/src/main/java/com/example/shin/debutcourandroid/MainActivity.java
package com.example.shin.debutcourandroid;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
   private Button btn ;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btn = (Button) findViewById(R.id.button);

        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Log.i("oc_rss", "ca marche");
                Intent myIntent = new Intent(MainActivity.this , next.class);
                startActivities(myIntent);
            }

            private void startActivities(Intent myIntent) {
                setContentView(R.layout.next);
            }
        });
    }

    
}

File: MyApplication/app/src/main/java/com/shin/myapplication/MainActivity.kt
package com.shin.myapplication

import android.app.Activity
import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v7.widget.Toolbar
import android.view.Menu
import android.view.MenuItem
import android.widget.Button
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var toolbar = findViewById(R.id.toolbar) as Toolbar
        setSupportActionBar(toolbar)
    }

    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
        menuInflater.inflate(R.menu.toolbar_main,menu)
        return true
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when(item.itemId){
            R.id.action_favorite->{
                val intent =Intent(this, FavoriteActivity::class.java)
                startActivity(intent)
                return true
            }
            R.id.action_delete->{
                Toast.makeText(this,"delete",Toast.LENGTH_SHORT).show()
                return true
            }
            else ->return super.onOptionsItemSelected(item)
        }

    }
}

